Userland/Utilities/unzip.cpp

```diff
@@ -72,18 +72,54 @@ static bool unpack_zip_member(Archive::ZipMember zip_member, bool quiet)
     return true;
 }
 
+static bool is_extract_match(StringView file_name, StringView filter_text)
+{
+    // FIXME: Collapse *'s and other tricks to reduce the footprint of the below lookup
+
+    // This serves as a calculated LUT of search comparisons
+    bool match_lookup[file_name.length()][filter_text.length()];
```

Comment:
Either way, VLAs are a no-no.

Reply:
Fix would be swapping for like a `Vector<>` implementation then?